portfolio/src/main/webapp/style.css

```diff
@@ -1,9 +1,51 @@
 #content {
   margin-left: auto;
   margin-right: auto;
+  justify-content: center;
```

Comment:
Nice use of flexbox!

nit: Alphabetize declarations within a rule (https://google.github.io/styleguide/htmlcssguide.html#Declaration_Order). This becomes much more important as rules grow to contain lots of declarations so you can quickly scan for the presence of an attribute. This applies to the rules below too but only leaving one comment on this here.